packages/db-postgres/src/queries/parseParams.ts

```diff
@@ -148,6 +148,10 @@ export async function parseParams({
                   break
                 }
 
+                if (field.type === 'relationship') {
+                  operator = 'equals'
```

Comment:
We can't just assume they want an `equals` operator. This wouldn't work if you were querying with `exists`, `in`, or `like`.